test/query_proto/bank-xunit.proto

```diff
@@ -0,0 +1,158 @@
+package sample_bank_account;

```

Comment:
To avoid messages like this during the build:

```
[libprotobuf WARNING google/protobuf/compiler/parser.cc:546] No syntax specified for the proto file: bank-xunit.proto. Please use 'syntax = "proto2";' or 'syntax = "proto3";' to specify a syntax version. (Defaulted to proto2 syntax.)
```

The proto files should contain a line like: `syntax = "proto3";` or `syntax = "proto2";`.